assignment-1-simple-calculator/calc.js

```diff
@@ -0,0 +1,61 @@
+
+var a = '';
+var b = '';
+var num = [];
+var ans;
+
+
+function sendNum(digit){
+
+	num.push(digit);
+
+	if(num.length != 1){
+		a = '';
+		document.getElementById('screen').innerHTML = a;
+	}
+
+
+	for(i=0; i<num.length ; i++){
+
+		a = a + num[i];
+
+	}
+
+document.getElementById('screen').innerHTML = a;
+
+
+}
+
+function equalTo(){
+
+	for(i=0; i<num.length ; i++){
+
+		b += num[i];
+	}
+
+	ans = eval(b);
```

Comment:
using eval, good choice, but lets make this complex.

Have you studied `Infix, Postfix and Prefix`, bring this up in our discussion.